app/views/projects/_form.html.erb

```diff
@@ -32,6 +32,11 @@
   </div>
 
   <div class="form-group">
+    <%= f.label :tag_group, class: 'control-label' %>
+    <%= f.select :tag_group_id, tag_group_options, {:include_blank => true}, { class: 'form-control auth-form-control' } %>
```

Comment:
The `+` button, to open a modal with a new tag group form here, like we talked before, is missing.